apps/i18n/dance/en_us.json

```diff
@@ -8,6 +8,21 @@
   "aiDanceFeedbackTinkerAiBlock": "Look at the new coding blocks that AI generated. Try choosing a new background or foreground effect.",
   "aiDanceNewMove": "You need to add a block to make the dancer do a new move.",
   "aiDanceStageChangeAfterMeasures": "Make sure to connect the `AI generate stage` block to the `After 4 Measures` event.",
+  "danceAiModalHeading": "generate {input} effect",
```

Comment:
We can probably use a technique like https://github.com/code-dot-org/ml-playground/pull/296 for this.

Reply:
On ai modal and block, 'effect' is plural ('effects'). Did you intend to change to singular?

Reply:
Oh! I see it's singular in updated UI.

Reply:
Yes, that's the new block text.